odoo/openupgrade/doc/source/modules50-60.rst

```diff
@@ -0,0 +1,342 @@
+Module coverage 5.0 -> 6.0
```

Comment:
But I have to keep the references to the files, and without the files the docs won't build locally.